README.md

```diff
@@ -1,2 +1,49 @@
 # PlanetParty
 Stretch Project in Mod 3 of Turing, learn/utilize Typescript
+
+# Getting Started with Create React App
```

Comment:
We can change all of this `README` stuff later. Looks like it's just the template for Create React App.